store/tikv/split_test.go

```diff
@@ -71,3 +71,30 @@ func (s *testSplitSuite) TestSplitBatchGet(c *C) {
 	err = snapshot.batchGetSingleRegion(s.bo, batch, func([]byte, []byte) {})
 	c.Assert(err, IsNil)
 }
+
+func (s *testSplitSuite) TestStaleEpoch(c *C) {
+	mockPDClient := &mockPDClient{client: s.store.regionCache.pdClient}
+	s.store.regionCache.pdClient = mockPDClient
+
+	firstRegion, err := s.store.regionCache.GetRegion(s.bo, []byte("a"))
+	c.Assert(err, IsNil)
+
+	txn := s.begin(c)
+	err = txn.Set([]byte("a"), []byte("a"))
+	c.Assert(err, IsNil)
+	err = txn.Set([]byte("c"), []byte("c"))
+	c.Assert(err, IsNil)
+	err = txn.Commit()
+	c.Assert(err, IsNil)
+
+	// Initiate a split and disable PD client. Now we can only update new
+	// region from kvrpc.
```

Comment:
Initiate a split and disable the PD client. If it still works, the new region is updated from kvrpc.